Project/Source/Components/ComponentTrail.cpp

```diff
@@ -37,6 +37,7 @@
 #define JSON_TAG_TRAIL_QUADS "TrailQuads"
 #define JSON_TAG_TEXTURE_REPEATS "TextureRepeats"
 #define JSON_TAG_QUAD_LIFE "QuadLife"
+#define JSON_TAG_IS_RENDERING "IsRendering"
```

Comment:
I think you don't want to serialize it since it's a value that depends on the status of the game, but If I'm wrong skip this comment

Reply:
I think I agree here